ST_structs.h

```diff
@@ -182,6 +184,10 @@ struct resourcegroup_st {
         max_density,  /* number of mature plants per plot allowed */
         max_per_sqm,  /* convert density and plotsize to max plants/m^2 */
         max_bmass,    /* sum of mature biomass for all species in group */
+        killfreq,       /* kill group at this frequency: <1=prob, >1=# years */
+        ignition,       /*cheatgrass biomass ignition in 1*1 square meter */
+        cheatgrass_coefficient,   /* cheatgrass biomass and fire frequency coeefficient */
+        wild_fire_slope,  /* slope of fire yearly possibility and cheatgrass biomass */	
```

Comment:
change to:

`/* slope of the cheatgrass biomass-wildfire probability relationship */`

Reply:
Commit 589665a implements this.